kafka-delete-topic/capability_service/kafka.py

```diff
@@ -0,0 +1,183 @@
+import logging
+import subprocess
+
+import requests
+from confluent_kafka.admin import AdminClient
+from requests import Response
+
+
+class InvalidKafkaCluster(Exception):
+    """Exception raised if trying to use a non-existing Kafka cluster."""
+
+    def __init__(
+        self,
+        environment,
+        message="Environment is not a valid Kafka cluster. Use 'dev' or 'prod'",
+    ):
+        self.environment = environment
+        self.message = message
+        super().__init__(self.message)
+
+    def __str__(self):
+        return f"{self.environment} -> {self.message}"
+
+
+class Topic:
+    """A class that represent a Kafka topic."""
+
+    def __init__(
+        self,
+        blaster_bearer_token: str,
+        blaster_topics_api_url: str,
+        blaster_cluster_uuid_prod: str,
```

Comment:
Is it necessary to have dublicated variables for prod and dev, couldn't they just be one which differ dependent on environment?